WordPress/UITests/Utils/XCTest+Extensions.swift

```diff
@@ -49,6 +49,8 @@ extension XCTestCase {
             app.launchArguments.append(contentsOf: ["-ui-test-select-wpcom-site", selectWPComSite])
         }
 
+        app.launchArguments.append(contentsOf: ["ff-override-Sidebar", "false"])
```

Comment:
I thought this line would magically disable Sidebar for UI tests.

Reply:
That was the idea. Not sure why it's not working.